sange/CLRS.tex

```diff
@@ -0,0 +1,54 @@
+\begin{sang}{CLRS}{Melodi: VLTJ - Tørfisk. Forfatter: Topholt}
+    \begin{vers}
+    Et sted derude mod store O
+    Derude hvor $n^2$ er go'
+    Kører nestet loops med rekursion
+    Mod nye mål med min og if
+    Mod nye mål med max og if
+    Den hedder CLRS
+    \end{vers}
+    
+    \begin{omkvaed}
+    Den starter med Insertion Sort
+    Går' over til run time analys'
+    Tar' hen mod bubble hvor den N'er helt i $O(n^2)$ //Kvadratisk
+    Alg'en er en dejlig hurtig op på stacken, ind i heapen, gennem queuen, når
+    Man læser i CLRS
+    \end{omkvaed}
+    
+    \begin{vers}
+    I de lumre lokaler langt ude mod øst
+    Læses bøgerne med manglende lyst
+    Dog læses bogen mer' og mer'
+    Læse alg på Cass-i-ope-ia
+    \textit{\$s du ved godt det ikke hedder Cass-i-ope-ia, det hedder Cass-i-opeia}
+    \textit{Det kan $(han||hun)$ da ikke selv gøre for (nå)}
+    Læse alg på Cass-i-opeia!
+    Med Bacci som lektor
+    \emph{Den starter med Insertion sort}\ldots
+    \end{vers}
+    
+    \begin{vers}
+    Og boolen kommer ud af loop
+    Et i-n i $\Theta$s navn
+    En dejlig run-time syns' de
+    Hvor alg'en vender og kører mod top
+    Hvor alg'en vender og kører mod top
+    Mod $O(log n)$ tid (Wrhh!) //logaritmisk
+    Var det seg fault?
+    \emph{Den starter med Insertion sort}\ldots
+    \end{vers}
+
+    \begin{vers}
+    Og endelig er array sortet
```

Comment:
Sortet og spået bør nok også rime bedre.

Måske noget i retningen af: "Et sorteret array har vi nu nået", og så lade anden linje være det samme.

Reply:
Tak for feedback, den passer ikke helt med stavelser, men måske "Sorteret arrray har vi nået" selvom det er klunky i slutningen imo. Jeg sover lige på den.